target/linux/bcm53xx/patches-5.4/030-ARM-dts-DLink-868L.patch

```diff
@@ -0,0 +1,154 @@
+
+--- /dev/null
++++ b/arch/arm/boot/dts/bcm4708-dlink-dir-868l.dts
+@@ -0,0 +1,139 @@
++/*
++ * Broadcom BCM470X / BCM5301X ARM platform code.
++ * DTS for D-Link DIR-868L
++ *
++ * Copyright (C) 2015 Rafał Miłecki <zajec5@gmail.com>
++ *
++ * Permission to use, copy, modify, and/or distribute this software for any
++ * purpose with or without fee is hereby granted, provided that the above
++ * copyright notice and this permission notice appear in all copies.
++ *
++ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
++ * REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
++ * AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
++ * INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
++ * LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE
++ * OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
++ * PERFORMANCE OF THIS SOFTWARE.
++ */
```

Comment:
Please use SPDX header and ideally `BCM5301X` standard `GPL-2.0-or-later OR MIT`. Feel free to drop my `Copyright` as there is pretty much nothing from my referenced code.